app/views/shared/_nav.html.erb

```diff
@@ -1,5 +1,7 @@
-<nav class="navbar navbar-expand-lg navbar-dark bg-dark">
-  <a class="navbar-brand" href="<%= landmarks_path %>">Oddio</a>
+<nav class="navbar navbar-expand-lg navbar-light bg-light">
```

Comment:
Let's try changing this to `navbar-expand-md` (instead of lg) so that the links show up without hitting the hamburger menu